.github/workflows/publish.yml

```diff
@@ -14,6 +14,10 @@ on:
   # daily schedule
   workflow_call:
 
+# TODO: remove once it's production
+env:
+  BASE_URL: /website-next/
```

Comment:
It has been 10 years that I have wanted to learn this setup:

<img width="813" alt="image" src="https://user-images.githubusercontent.com/11404065/217092866-a377c7ff-8baf-4042-807c-824a4eb99a42.png">


I think it is the right time 😄

Reply:
go for it:rocket: :rocket: can imagine the high levels of energy with which you'll be setting it up:fire::fire: